trade-finance-manager-api/.eslintrc.js

```diff
@@ -18,7 +18,15 @@ const baseRules = {
   'import/no-extraneous-dependencies': [
     'error',
     {
-      devDependencies: ['**/*.test.{js,ts}', '**/*.api-test{,s}.{js,ts}', '**/*.spec.{js,ts}', '**/webpack.*.{js,ts}', '**/api-test*/**', '**/__mocks__/**'],
+      devDependencies: [
+        '**/*.test.{js,ts}',
+        '**/*.api-test{,s}.{js,ts}',
+        '**/*.spec.{js,ts}',
+        '**/webpack.*.{js,ts}',
+        '**/api-test*/**',
+        '**/*.tests.{js,ts}',
```

Comment:
What's the reason for adding `*.tests`, can we just use the existing `*.test`? Otherwise we could end up with some files using ".test", other ".tests"

Reply:
We're using `tests` to contain reusable tests that are not standalone and only run when imported (if you search for `with-` you'll find several test files which contain reusable tests that do not run when running `npm run unit-test-dev` unless imported into other files)